app/controllers/oidc_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class OidcController < RestController
```

Comment:
Let's get this into an `Authenticators` namespace to denote its purpose.  This is accomplished by moving it into the folder `app/controllers/authenticators` and moving the above to:

```ruby
module Authenticators
  class OidcController < RestController
    ...
```

Reply:
i like it, i made the changes to this new service.